client/my-sites/stats/activity-log/style.scss

```diff
@@ -90,30 +96,29 @@
 	.activity-log__threat-alert-title {
 		color: black;
 	}
+
+	.activity-log__threat-alert-title-slug {
+		font-weight: bold;
+	}
 }
 
-.activity-log__threat-alert-file-header {
-	margin-top: 1em;
-	margin-bottom: 4px;
+.activity-log__threat-alert-signature {
+	font-weight: bold;
+}
 
-	.activity-log__threat-alert-filename {
-		color: black;
-		font-weight: 500;
-	}
+.activity-log__threat-alert-filename {
+	padding: 4px;
+	color: black;
```

Comment:
This should be changed to #000 because this is not converted when it's built and it's wasting 1 byte. Otherwise it should use some dark color like $gray-dark https://dotcombrand.wordpress.com/colors/

Reply:
will save the internet 1 byte 😄